src/ogdc_runner/recipe/shell.py

```diff
@@ -88,48 +99,6 @@ def make_and_submit_shell_workflow(
     return workflow_name
 
 
-def submit_ogdc_recipe(
```

Comment:
This function living in `shell.py` is what gave rise to #85 and this PR. It is not specific to the `shell` recipe type - it's used by the `viz_workflow` as well! Moved to `api`.